presto-docs/src/main/sphinx/release/release-0.166.rst

```diff
@@ -0,0 +1,10 @@
+=============
+Release 0.166
+=============
+
+General Changes
+---------------
+
+* Fix failure due to implicit coercion issue in ``IN`` expressions for certain combinations of data types (e.g, ``double`` and ``decimal``).
+* Add `query.max-length`config flag to set the maximum length of a SQL query. The default maximum length is 1MB.
```

Comment:
Wrap the second sentence. Code formatting needs double backticks. Missing space before "config flag"